dev/Dockerfile

```diff
@@ -10,9 +10,11 @@ ARG DISABLE_SPRING=1
 RUN apk add --update --no-cache \
     build-base \
     postgresql-dev \
```

Comment:
Looks good! Can you actually also remove this
`RUN apt-get update && apt-get install libjemalloc1 && rm -rf /var/lib/apt/lists/*
ENV LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so.1` ?

Simon and me used this for testing once and forgot to remove it! Thank youuuu